### PR TITLE
I18nText widget update

### PR DESCRIPTION
**Pull request by tvhenrik**
Found it useful and less boilerplaty to have a translated text widget, maybe others will too :)

**New Features:**
- I18nText widget
  - Supports translate with params and plural

**Changes:**
- changed root -gitignore to include dart/flutter files
  - will be useful for future testing suites
- removed usage of `new` in example in favor of the more common `new`less syntax
- changed `Map.fromIterables(["user"], ["Flutter lover"])` to `{"user": "Flutter lover"}` in example
- added I18nText usage examples in readme

### CODE REVIEW

File: lib/i18n_text.dart [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+  const I18nText(
+    this.locKey, {
+    this.params,
+    this.plural,
+    Key key,
+    this.style,
+    this.strutStyle,
+    this.textAlign,
+    this.textDirection,
+    this.softWrap,
```
Comment (ilteoood) [inline lib/i18n_text.dart]: All of these properties are copied from the text widget, right?
What if you use the composition and pass the text widget as child parameter? With this approach, you don't depend on the implementation of the text widget

Comment (tvhenrik) [reply lib/i18n_text.dart]: Yep, it's an ugly copy paste from the text widget solution.
I'm not sure what you mean, passing a text widget as child would mean it's text is already set?

Comment (ilteoood) [reply lib/i18n_text.dart]: Yes, it is.
You can use the passed Text widget as DTO in order to transfer all these fields; then, in the build method, you can create and return a new Text widget copying all the fields except for data, whose value will be retrieved from the translation process.

**Comment by ilteoood** [comment]
I've reworked your idea and published the result with the release 0.10.0.

Thank you so much for your idea!

**Pull request closed without merging**